restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -361,9 +362,9 @@
 	"projectLogResetLabel": "Reiniciar",

```

Comment:
There's an error in (340). Change for:
"multibuttonButton": "Seleccione la opción desplegable",